LICENSE.md

```diff
@@ -71,3 +72,28 @@ and .travis.yml contain code copied from the Transducers.jl package
 >LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 >OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 >SOFTWARE.
+
+The `TreePrint` module contains code from the AbstractTrees.jl package
```

Comment:
I don't think this is needed, `AbstractTrees` is a dependency, you did not copy code directly right?

Reply:
Yes, I did copy code directly in the `TreePrint` module, just to use https://github.com/Keno/AbstractTrees.jl/pull/38 on our side. We can delete the module and the copied code once that PR is merged, but I don't think it will be soon since the repo seems pretty inactive.